### ISSUE TITLE
Cannot Open data files after processing

### ISSUE THREAD

**Issue by JPerAsperaadAstra** [ristea/aed-mae#8]
Hi, dear authors. Thanks for your outstanding work. And here's a problem I encountered after processing the dataset following README:
I created anomaly sequces from UBnormal dataset and integrated them with util/create_anomalies/create_abnormal_sequences.py and util/create_anomalies/integrate_abnormal_objects.py, then I organited the dataset as following structure:
-Avenue
--testing
---frames
---gradients
--traing
---frames
---frames_abnormal
---gradients
---masks_abnormal
where frames_abnormal and masks_abnormal were generated with pre-processing scripts util/create_anomalies/main.py, gradients was generated by extract_gradients.py, and frames was the original frames from avenue dataset. 
However, when I ran main.py on the avenue dataset, I got following errors:
[ WARN:0@14.611] global loadsave.cpp:248 findDecoder imread_('datasets/Avenue/training/frames_abnormal/07/541.jpg'): can't open/read file: check file path/integrity
 
It seems to be a data format or integrity problem, but I was able to open this jpg just fine. Could you please give me some suggestion about fixing this error and make it work? Thx

**Comment by CroitoruAlin** [comment]
Hello! This looks like something related to your environment. I see that you are using a relative path, please make sure that it is correct.

**Comment by JPerAsperaadAstra** [comment]
Hi, thx for your quick response. I tried to use the absolute path but resulted in the same error. Could it be possible that my extracted frames of avenue dataset is .jpg while abnormal sequences are save as .png?

**Comment by JPerAsperaadAstra** [comment]
> [pleted](/ristea/aed-mae/issues?q=is%3Aissue+is%3Aclosed+archived%3Afalse+reason%3Acompleted) [May 26, 2024](#event-12937040243)

Hi, thx for your quick response. I tried to use the absolute path but resulted in the same error. Could it be possible that my extracted frames of avenue dataset is .jpg while abnormal sequences are saved as .png? (since frames_abnormal are abnormal_sequences added to normal data)

**Comment by houchenfeng** [comment]
Image name is wrong, you can check the datasets/Avenue/training/frames_abnormal / 07,
I guess the image under the path should be "0541.jpg". Therefore, modify the parameters such as len_frame_no of the _getitem_ function in data/train_dataset.py .Or modify the image format for masks_abnormal in about 48 lines.

**Comment by JPerAsperaadAstra** [comment]
@flww213 Thank you so much and your suggestion is right!
